web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/util/AESEncrypter.java

```diff
@@ -24,44 +24,49 @@
 import javax.crypto.Cipher;
 import javax.crypto.spec.IvParameterSpec;
 import javax.crypto.spec.SecretKeySpec;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 
+@Component
 public class AESEncrypter {
 
-    private static final String KEY = "0123456789abcdef";
+    private static final Logger LOG = LoggerFactory.getLogger(AESEncrypter.class);
     private static final String INIT_VECTOR = "fedcba9876543210";
+    @Value("${aes.encryption.key}")
+    private String key;
 
-    public static String encryptPatientId(String value) {
+    public String encryptPatientId(String value) {
```

Comment:
Maybe could refactor the class to reuse the cipher object. One for decrypt and one for encrypt. 

Also consider the name of the class to something more specific. 

Would be great to reintroduce some unit tests. You can use the @Value in the constructor and initialise it for test with different keys.